cli/dcoscli/tables.py

```diff
@@ -145,9 +149,20 @@ def deployment_table(deployments):
 
     """
 
+    def resource_path_id(action):
+        path_id = action.get('app') or action.get('pod')
+
+        if path_id is None:
+            logger.exception('Missing app or pod entry')
+
+            template = 'Expected "app" or "pod" field in action: {}'
+            raise ValueError(template.format(action))
```

Comment:
So if any of the actions in a particular deployment have neither `"app"` nor `"pod"`, set the `ACTION` column for that row to `N/A`?